doc/en/user/source/community/gwc-mbtiles/index.rst

```diff
@@ -0,0 +1,50 @@
+.. _community_gwc_mbtiles:
+
+GWC MBTiles layer plugin
+========================
+
+This plugin allows to adding a GWC MBTiles Layer to GeoServer's cached Tile Layers configuration so that GeoServer can load cached tiles from ready-to-use MBTiles.
+
+Installation
+------------
+
+As a community module, make user you have donwloaded the :download_extension:`gwc-mbtiles`
```

Comment:
```suggestion
As a community module, make sure you have downloaded the :download_extension:`gwc-mbtiles`
```

Reply:
Ah ah. I have copied that line from another module already using the download_extension MACRO.
Didn't notice that :).
Fixing it.